doc/api/blivet.rst

```diff
@@ -67,7 +65,6 @@ blivet
         * :attr:`~blivet.blivet.Blivet.swaps`
         * :attr:`~blivet.blivet.Blivet.thinlvs`
         * :attr:`~blivet.blivet.Blivet.thinpools`
-        * :meth:`~blivet.blivet.Blivet.update_ksdata`
         * :attr:`~blivet.blivet.Blivet.vgs`
```

Comment:
That's an interesting point; I'm planning to merge this into blivet-3.0, even though eventually this is going to all get ripped out and moved into anaconda (it's just living here for the interim). I'm not really sure what the timeline for that is though.